locales/en.json

```
@@ -1,6 +1,143 @@
+      "visitor": "Visitor",
+      "speaker": "Speaker",
+      "admin": "Admin"
+    }
+  },
+  "EventManagement": {
+    "title": "Events Management",
+    "addNewEvent": "Add New Event",
+    "eventsCount": "Events",
+    "noDescription": "No description",
+    "capacity": "Capacity",
+    "noPresenter": "No Presenter",
```

Biome (2.1.2)

[error] The key capacity was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)